cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -200,6 +202,18 @@ func getAPIServerCommand(cfg *kubeadmapi.ClusterConfiguration, localAPIEndpoint
 	if cfg.APIServer.ExtraArgs == nil {
 		cfg.APIServer.ExtraArgs = map[string]string{}
 	}
+
+	if audiences := cfg.APIServer.ExtraArgs["api-audiences"]; audiences != "" {
+		defaultArguments["api-audiences"] = audiences
```

Comment:
this is not required. later manifests.go merges the default arguments and the user arguments (from extraArgs), where extraArgs take precedence.

Reply:
oh, thanks. done